go.mod

```diff
@@ -5,7 +5,16 @@ go 1.12
 require (
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
+	github.com/pkg/errors v0.8.1
+	github.com/submariner-io/admiral v0.0.0-20190717183806-7f0ad3dca605
 	github.com/submariner-io/submariner v0.0.0-20190708095718-350482d85dd4
+	k8s.io/api v0.0.0-20190222213804-5cb15d344471
+	k8s.io/apimachinery v0.0.0-20190629003722-e20a3a656cff
+	k8s.io/client-go v0.0.0-20190521190702-177766529176
+	k8s.io/klog v0.0.0-20181108234604-8139d8cb77af
 )
 
-replace github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190223031456-9a5ae4453bd
+replace (
+	github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190223031456-9a5ae4453bd
+	github.com/submariner-io/admiral => github.com/tpantelis/admiral v0.0.0-0.20190715192543-8c6db85
```

Comment:
Getting the following error when compiling.

go: github.com/tpantelis/admiral@v0.0.0-0.20190715192543-8c6db85: unknown revision 8c6db85
go: error loading module requirements
FATA[0009] exit status 1                                
make: *** [Makefile:11: test] Error 1

Reply:
This is no longer needed as my branch has been merged into the admiral master branch.